Docker/worker_base/Dockerfile

```diff
@@ -43,12 +32,18 @@ RUN apt-get update && apt-get install -y \
       dotnet-sdk-3.1 \
       dotnet-sdk-5.0 \
       openjdk-11-jdk \
-      nodejs \
       python3.8 \
       python3-pip \
       golang-go \
       libgtk-3-0 \
       libgbm1 && \
+      mkdir -p /etc/apt/keyrings; \
+       curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key \
+     | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg; \
+    echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_20.x nodistro main" \
+     > /etc/apt/sources.list.d/nodesource.list; \
```

Comment:
I think this can be done before this RUN command and here just add `nodejs` along with the others.